packages/react/src/components/Toggletip/index.tsx

```diff
@@ -225,9 +225,9 @@ Toggletip.propTypes = {
   ]),
 
   /**
-   * Provide an offset value for alignment axis.
+   * Will add padding value to arrow in case of bottom-left, bottom-right or top-left, top-right
    */
-  alignmentAxisOffset: PropTypes.number,
+  arrowPadding: PropTypes.number,
```

Comment:
Mirror the `caretPadding` name change and updated prop/type comment from Popover to here